libraries/AC_PID/AC_PID.cpp

```diff
@@ -253,8 +334,8 @@ float AC_PID::get_d() const
 
 float AC_PID::get_ff()
 {
-    _pid_info.FF = _target * _kff;
-    return _target * _kff;
+    _pid_info.FF = _target * _kff + _target_derivative * _kdff;
```

Comment:
This is a existing bug but this but of the calculation should be done in the main `update_all` function. This removes a race condition between when we update, when we log and when we get the ff.

Reply:
Fixed.